addons/l10n_mx/data/account_tag_data.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" ?>
 <odoo>
-    <data noupdate="1">
+    <data noupdate="0">
```

Comment:
Yes @moylop260 

The user not must be change this values in the [System](https://github.com/Vauxoo/mexico/blob/10.0/l10n_mx_base/models/account.py#L115), if we need update a value, must be by data.

With update=1, we can not update from XML.